### PR TITLE
[LOG] print output dim instead of input dim in model summary  @open sesame 06/13 14:48

### PR DESCRIPTION
**Pull request by baek2sm**
When we print the model architecture using summarize method, nntrainer prints input dimension of each layer.

But, tensorflow and pytorch are printing output dimmension of each layer in the summary, thus it is inconvenient to compare each layer with tf and torch models.

Thus, I suggest to print output dimension of each layer instead of input dimension in the model summary.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM

**Review by myungjoo — APPROVED** [review]

**Review by songgot — APPROVED** [review]
LGTM
